css/style.css

```diff
@@ -28,17 +28,23 @@ body::before
 }
 
 /* show signature download on hover */
-span.sig_btn > a:last-child
+span.sig_btn
 {
+    position: relative;
+    height: 4.4em;
```

Comment:
Dunno, I think opacity has a bigger performance impact on rendering. I'll check if I can avoid the hard-coded height.